index.html

```diff
@@ -16,20 +18,27 @@
 <body>
   <header>
   <nav>
-    <ul class="menu">
-        <div>
-          <li class="logo"><a href="">My Logo</a></li>
-        </div>
-        <div class="menu2">
-          <li class="item"><a href="#">Portfoilo</a></li>
-          <li class="item"><a href="#">about</a></li>
-          <li class="item"><a href="#">contact</a></li>
-          <li class="toggle"><a href=""><img src="./images/Icon - Menu.png" alt="humberger menu image"></a></li>
-        </div>
-    </ul>
+    <div class="menu">
+          <div class="logo"><a href="#">My Logo</a></div>
+        <ul class="desktop-nav st1">
+          <li class="nav-link"><a href="#portfolio">Portfolio</a></li>
+          <li class="nav-link"><a href="#about">about</a></li>
+          <li class="nav-link"><a href="#contact">contact</a></li>
+        </ul>   
+        <ul class="menu2 st">
+          <li class="item"><a href="#portfolio">Portfolio</a></li>
+          <li class="item"><a href="#about">about</a></li>
+          <li class="item"><a href="#contact">contact</a></li>
+        </ul>       
+        <ul class="breadcrumb">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </ul> 
+    </div>
```

Comment:
Nice work implementing the functionalities of the design👍. Your mobile menu looks a bit different from the given design🥲 Kindly see the comparison below and ensure your mobile menu matches the design as it is a requirement that we stick to the given design in this milestone👍.

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="400" alt="your mobile" src="https://user-images.githubusercontent.com/64643995/233347056-345ba8f8-90f4-4dd2-8b20-8b816c777ff4.png">|<img width="400" alt="design" src="https://user-images.githubusercontent.com/64643995/233347159-a1ccafe1-49f0-41b2-965b-31e507b3a77b.png">|

Kindly take note of the position of the elements, font properties and the background blur😉